build.gradle

```diff
@@ -313,6 +313,12 @@ project(':core') {
     from sourceSets.test.output
   }
 
+  task publishDocs(type: Jar) {
```

Comment:
@ijuma  We are not discussing about javadocs.  We want  kafka/site documentation pages to be part of release of artifacts.